Source/WebCore/dom/WritingSuggestionData.h

```diff
@@ -46,10 +49,20 @@ class WritingSuggestionData {
 
     uint64_t offset() const { return m_offset; }
 
+    String originalPrefix() const { return m_originalPrefix; }
+
+    String originalSuffix() const { return m_originalSuffix; }
+
+    bool supportsSuffix() const { return m_supportsSuffix; }
+
 private:
     String m_content;
     String m_currentText;
     uint64_t m_offset;
+
+    String m_originalPrefix;
+    String m_originalSuffix;
+    bool m_supportsSuffix;
```

Comment:
Nit - might be good to give this an initial value to prevent UB in the future, though I think it's okay either way (up to you!).

Same regarding `m_offset` above